pennylane/ops/qubit.py

```diff
@@ -486,21 +538,38 @@ class CRY(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
-    generator = [np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, -1j], [0, 0, 1j, 0]]), -1 / 2]
+    generator = [
+        np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, -1j], [0, 0, 1j, 0]]), -1 / 2]
+
+    @staticmethod
+    def decomposition(theta, wires):
+        decomp_ops = [
+            U3(theta / 2, 0, 0, wires=wires[1]),
+            CNOT(wires=wires),
+            U3(-theta / 2, 0, 0, wires=wires[1]),
+            CNOT(wires=wires)
+        ]
+        return decomp_ops
 
 
 class CRZ(Operation):
     r"""CRZ(phi, wires)
     The controlled-RZ operator
 
-    .. math:: CR_z(\phi) = \begin{bmatrix}
+    .. math::
+        \begin{align}
+             CRZ(\phi) &= I_{1}\otimes PhaseShift_{2}(\pi / 2) ~\cdot~ CNOT_{12} ~\cdot~ I_{1}\otimes PhaseShift_{2}(-\pi / 2) ~\cdot~ CNOT_{12} \notag \\[10pt]
```

Comment:
Better to write the phase shift operation in math notation (`R_{\phi}(\pi/2)`)

Reply:
As per our discussion, we'll update this one as well, once we've come up with a new notation for the `PhaseShift`